__mocks__/fs-extra.js

```diff
@@ -0,0 +1,66 @@
+const fs = require("fs");
+const path = require("path");
+
+exports.remove = jest.fn();
```

Comment:
This file is a mock of `fs-extra`. It makes it so that we can run packmap without having to actually access the file system at all.